app/recordtransfer/static/recordtransfer/css/base/widget.css

```diff
@@ -41,4 +41,4 @@
     .form-tooltip {
         width: 200px;
     }
-}
+}
```

Comment:
Avoid committing changes to files unnecessarily - this one is very minor so we can ignore it but try to only include changes that are relevant to the issue.